recipes/tn93/meta.yaml

```diff
@@ -1,9 +1,9 @@
 package:
   name: tn93
-  version: 1.0.2
+  version: 1.0.4
 
 source:
-  git_rev: v1.0.2
+  git_rev: v1.0.4
   git_url: https://github.com/veg/tn93.git
```

Comment:
Sorry Bjorn, 
I just used tn93's current configuration for this pull request. I can make the recommended updates.